api/lib/reducers/polkadotV0-reducers.js

```diff
@@ -516,14 +516,16 @@ function depositReducer(deposit, depositer, network) {
   }
 }
 
-function networkAccountReducer(account) {
+function networkAccountReducer(account, store) {
+  const validator = account ? store.validators[account.accountId] : undefined
   return {
     name:
       account && account.identity && account.identity.display
         ? account.identity.display
         : undefined,
     address: account && account.accountId ? account.accountId : '',
-    picture: account ? account.twitter : '' // TODO: get the twitter picture using scriptRunner
+    picture: validator ? validator.picture : account ? account.twitter : '',
```

Comment:
What does this mean? Did you change something to get that Twitter image?

Reply:
the validators are already enriched with pictures so we use them from there

Reply:
ah ok. And the Polkadot councellors we can get them later

Reply:
ah, now I see. Yes, I think I was just sleepy :sweat_smile: